diff --git a/lustre/tests/sanity.sh b/lustre/tests/sanity.sh
index 0ac6fb9ff..4c4c1ae30 100644
--- a/lustre/tests/sanity.sh
+++ b/lustre/tests/sanity.sh
@@ -49,11 +49,11 @@ if $SHARED_KEY; then
 	ALWAYS_EXCEPT+=" 17n     60a     133g    300f    272a"
 fi
 
-selinux_status=$(getenforce)
-if [ "$selinux_status" != "Disabled" ]; then
+#selinux_status=$(getenforce)
+#if [ "$selinux_status" != "Disabled" ]; then
 	# bug number:    LU-12469 LU-12469
-	ALWAYS_EXCEPT+=" 230b     230d"
-fi
+#	ALWAYS_EXCEPT+=" 230b     230d"
+#fi
 
 # skip the grant tests for ARM until they are fixed
 if [[ $(uname -m) = aarch64 ]]; then
